backend/packages/Upgrade/src/api/services/SegmentService.ts

```diff
@@ -36,6 +37,11 @@ interface IsSegmentValidWithError {
   isSegmentValid: boolean;
 }
 
+interface SegmentParticipantsRow {
+  UUID: string;
```

Comment:
This results in the header for the segment (or user) name being 'UUID'. At least in the case of segments, it's not likely to be a UUID. Maybe 'Name' would be a better header text?

Reply:
true but csv shared by april and zack contains UUID so I kept it that way.